packages/devtools_app/lib/src/screens/network/network_request_inspector_views.dart

```diff
@@ -146,10 +146,22 @@ class HttpRequestView extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    final theme = Theme.of(context);
+    final requestHeaders = data.requestHeaders;
+    final requestContentType = requestHeaders != null && requestHeaders.containsKey("content-type") ? requestHeaders["content-type"] : "";
```

Comment:
Have you considered replacing this with:

```dart
final requestContentType = requestHeaders?['content-type'] ?? '';
```

Also, you'll need to use single quotes in general, otherwise our analysis checks will block the PR.

Reply:
Updated as per comment